.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and push Docker image
```

Comment:
These changes should be applied to the existing `build-publish-chainlink.yml` file